src/test/prevector_tests.cpp

```diff
@@ -198,67 +198,65 @@ BOOST_AUTO_TEST_CASE(PrevectorTestInt)
     for (int j = 0; j < 64; j++) {
         prevector_tester<8, int> test;
         for (int i = 0; i < 2048; i++) {
-            int r = insecure_rand();
-            if ((r % 4) == 0) {
+            if (insecure_randbits(2) == 0) {
                 test.insert(insecure_randrange(test.size() + 1), insecure_rand());
             }
-            if (test.size() > 0 && ((r >> 2) % 4) == 1) {
+            if (test.size() > 0 && insecure_randbits(2) == 1) {
```

Comment:
In commit "Use randbits instead of ad-hoc emulation in prevector tests"

Can all these `== 1`, `== 2`, `== 3` checks be replaced with `== 0`? That would seem like a simpler style.